### ISSUE TITLE
Last commit 2015, is this project dead?

### ISSUE THREAD

**Issue by therealmarv** [ioerror/tlsdate#199]
See the question on topic.

**Comment by oniGino** [comment]
Yeah its too bad, but pretty sure this is dead, since most modern TLS servers no longer send time but instead send random numbers..

Here is a quick blurb that sums it up well
https://security.stackexchange.com/questions/71364/tls-reliance-on-system-time

**Comment by therealmarv** [comment]
ok, thanks a lot for the info and link!

**Comment by vapier** [comment]
@ioerror: CrOS uses this project quite a bit and continues to maintain it.  would you be OK with having some of us help out with maintenance here ?

obviously we wouldn't turn this into some CrOS-only project, but merge fixes/general improvements that make sense everywhere.

**Comment by ioerror** [comment]
Yes, that would be welcomed. I've reviewed a number of CrOS patches
and every one of them seemed worth landing. Please open pull requests,
I'll review and merge as appropriate. Thank you for the offer of help.

**Comment by vapier** [comment]
i was thinking you'd add a few of us (security minded peeps) as Members and then we'd be able to review/merge changes directly in this particular repo

**Comment by ioerror** [comment]
On 8/21/18, Mike Frysinger <notifications@github.com> wrote:
> i was thinking you'd add a few of us (security minded peeps) as Members and
> then we'd be able to review/merge changes directly in this particular repo

Interesting thought. Please send me an email.

**Comment by asarubbo** [comment]
I just want to let you know that since there is no activity, the package was masked and will be removed on the gentoo repository unless there will be activities again..
@ioerror since the package is maintained for chrome os, what's the progress to add the CrOS people to this repository?
TIA

**Comment by ioerror** [comment]
On 11/4/18, Agostino Sarubbo <notifications@github.com> wrote:
> I just want to let you know that since there is no activity, the package was
> masked and will be removed on the gentoo repository unless there will be
> activities again..

Is Gentoo carrying any patches that need to be merged?

> @ioerror since the package is maintained for chrome os, what's the progress
> to add the CrOS people to this repository?

I haven't received an email from them yet.

**Comment by asarubbo** [comment]
> Is Gentoo carrying any patches that need to be merged?
In our case, atm the tlsdated daemon crashes (because of glibc changes), so we have two ways:
- find patches around the world
- find patches in the official repo and/or make a snapshot which contains the patch and/or obtain a new release.

Usually we prefer the latest.

> @ioerror since the package is maintained for chrome os, what's the progress to add the CrOS people to this repository?
> I haven't received an email from them yet.
@vapier any news?

**Comment by dkriegner** [comment]
> > Is Gentoo carrying any patches that need to be merged?

In 2015 I opened a pull request with a gentoo patch: #171 
There was no feedback whatsoever since then!

**Comment by orent** [comment]
Note to anyone who comes here:

There is an active fork by the ChromiumOS project here:

https://chromium.googlesource.com/chromiumos/third_party/tlsdate

In general, it appears to be a bad idea to rely on TLS handshake using the timestamp, rather than some random data. 

But since the Chromium project is by Google there is probably some long term commitment that at least TLS services on google.com continue supporting it.
